### PR TITLE
Update docs to Windows 11 

### PR DESCRIPTION
**Pull request by noraa-junker**
* hub/apps/design
* hub/apps/desktop
* hub/apps/get-started
* hub/apps/performance

> This PR is part of my effort to bring all Microsoft Docs to Windows 11. You can see my overall progress in [this gist](https://gist.github.com/Aaron-Junker/ad8f6234f1a991e46e234b25dc0ce0dc)

### CODE REVIEW

**Comment by ShannonLeavitt** [comment]
#label:"aq-pr-triaged"

**Comment by noraa-junker** [comment]
@ktoliver The images of the mail app in [hub/apps/design/in-app-help/images](https://github.com/Aaron-Junker/windows-uwp/tree/win10to11-4/hub/apps/design/in-app-help/images) need an update. Could you create them?

**Comment by jborsecnik** [comment]
> Aaron-Junker

Hi @Aaron-Junker. I'm sorry, but the PR review team can't currently do this.

File: hub/apps/design/globalizing/design-for-bidi-text.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Four elements influence BiDi design decisions in Windows.
 
 ## Feature design overview
 
-Windows supports the four BiDi design elements. Let's look at some of the major relevant features in Windows, and provide some context around how they affect your app.
+Windows supports the four BiDi design elements. Let's look at some of the major relevant features of (prior versions of) Windows, and provide some context around how they affect your app.
```
Comment (Jay-o-Way) [inline hub/apps/design/globalizing/design-for-bidi-text.md:34]: Prior to what? Is this not current?

Comment (noraa-junker) [reply hub/apps/design/globalizing/design-for-bidi-text.md:34]: The screenshots describe sometimes the tiles in Windows 8

File: hub/apps/design/input/access-keys.md [hunk 2]
```diff
@@ -113,7 +113,7 @@ _CommandBar primary scope and supported access keys_
 
 _CommandBar secondary scope and supported access keys_
 
-### Windows 10 Creators Update and older
+### Windows 10 Creators Update and newer
```
Comment (Jay-o-Way) [inline hub/apps/design/input/access-keys.md]: Why is this the opposite? What is the intention of this part?

File: hub/apps/design/style/segoe-ui-symbol-font.md [hunk 3]
```diff
@@ -59,7 +59,7 @@ You can also use the static resource `SymbolThemeFontFamily` to access **Segoe M
 
 
 ## How do I get this font?
-* On Windows 10: There's nothing you need to do, the font comes with Windows.
+* On Windows: There's nothing you need to do, the font comes with Windows.
```
Comment (Jay-o-Way) [inline hub/apps/design/style/segoe-ui-symbol-font.md:62]: Are you sure MDL2 is still included in the latest version of a Win11 installation?

File: hub/apps/desktop/modernize/modernize-wpf-tutorial-4.md [hunk 4]
```diff
@@ -8,16 +8,19 @@ ms.localizationpriority: medium
 ms.custom: RS5, 19H1
 ---
 
-# Part 4: Add Windows 10 user activities and notifications
+# Part 4: Add Windows user activities and notifications
 
 This is the fourth part of a tutorial that demonstrates how to modernize a sample WPF desktop app named Contoso Expenses. For an overview of the tutorial, prerequisites, and instructions for downloading the sample app, see [Tutorial: Modernize a WPF app](modernize-wpf-tutorial.md). This article assumes you have already completed [part 3](modernize-wpf-tutorial-3.md).
 
-In the previous parts of this tutorial you added UWP XAML controls to the app using XAML Islands. As a by product of this, you also enabled the app to call any WinRT API. This opens up the opportunity for the app to use many other features offered by Windows 10, not just UWP XAML controls.
+In the previous parts of this tutorial you added UWP XAML controls to the app using XAML Islands. As a by product of this, you also enabled the app to call any WinRT API. This opens up the opportunity for the app to use many other features offered by Windows, not just UWP XAML controls.
```
Comment (Jay-o-Way) [inline hub/apps/desktop/modernize/modernize-wpf-tutorial-4.md]: by-product or byproduct

File: hub/apps/desktop/modernize/modernize-wpf-tutorial-4.md [hunk 5]
```diff
@@ -8,16 +8,19 @@ ms.localizationpriority: medium
...
 
 In the fictional scenario of this tutorial, the Contoso development team has decided to add two new features to the app: activities and notifications. This part of the tutorial shows how to implement these features.
 
 ## Add a user activity
 
+>[!Note]
+> The timeline feature is discontinued since Windows 11
```
Comment (Jay-o-Way) [inline hub/apps/desktop/modernize/modernize-wpf-tutorial-4.md:22]: Doesn't that mean this entire section is deprecated now?

File: hub/apps/get-started/developer-mode-features-and-debugging.md [hunk 6]
```diff
@@ -127,9 +127,9 @@ To fix this issue:
...
+For most developers, you want to use the settings app to enable your device for debugging. In certain scenarios, such as automated tests, you can use other ways to enable your Windows desktop device for development.  Note that these steps will not enable the SSH server or allow the device to be targeted for remote deployment and debugging.
 
-You can use gpedit.msc to set the group policies to enable your device, unless you have Windows 10 Home. If you do have Windows 10 Home, you need to use regedit or PowerShell commands to set the registry keys directly to enable your device.
+You can use gpedit.msc to set the group policies to enable your device, unless you have Windows 10 Home ore Windows 11 Home. If you do have Windows 10 Home or Windows 11 Home, you need to use regedit or PowerShell commands to set the registry keys directly to enable your device.
```
Comment (Jay-o-Way) [inline hub/apps/get-started/developer-mode-features-and-debugging.md]: ore ➡️ or

File: hub/apps/performance/choose-between-tools.md [hunk 7]
```diff
@@ -26,7 +26,7 @@ There is overlap between the two tools. Often you can start exploring an issue i
 
 The Visual Studio Performance Profiler is a component of Visual Studio itself and is installable through the same [installation wizard](/visualstudio/install/install-visual-studio) as the rest of the development environment.
 
-The Windows Performance Toolkit is downloadable separately as a part of the [Windows Assessment and Deployment Kit](/windows-hardware/get-started/adk-install). However, the command-line Performance Recorder tool is pre-installed with Windows 10 and later as `wpr.exe` available on the default path variable from shells like PowerShell and the Command Prompt.
+The Windows Performance Toolkit is downloadable separately as a part of the [Windows Assessment and Deployment Kit](/windows-hardware/get-started/adk-install). However, the command-line Performance Recorder tool is pre-installed with Windows 10 and Windows 11 and later as `wpr.exe` available on the default path variable from shells like PowerShell and the Command Prompt.
```
Comment (Jay-o-Way) [inline hub/apps/performance/choose-between-tools.md]: This specific change is kind of redundant. Win11 _is_ later then 10, and there's nothing later (yetl

File: hub/apps/design/input/access-keys.md (hunk 2 above)
Comment (noraa-junker) [reply hub/apps/design/input/access-keys.md]: Was a mistake

File: hub/apps/desktop/modernize/modernize-wpf-tutorial-4.md (hunk 5 above)
Comment (noraa-junker) [reply hub/apps/desktop/modernize/modernize-wpf-tutorial-4.md:22]: No. It is still curreent in Windoows 10 which is actively supported.

File: hub/apps/design/style/segoe-ui-symbol-font.md (hunk 3 above)
Comment (noraa-junker) [reply hub/apps/design/style/segoe-ui-symbol-font.md:62]: Yes. It is insluded.

**Comment by noraa-junker** [comment]
@mattwojo What does this, #3705 and MicrosoftDocs/WSL#1482 hold from merging?

**Comment by GrantMeStrength** [comment]
Thanks Aaron!

**Pull request merged**